File: Python/Lista 4/lowestNumberPosition.py
L = []
n = int(input())
posi = menor = 0
L = input().split()
for i in range(n):
    L[i] = int(L[i])
    if i == 0:
        menor = L[i]
        posi = i
    if i != 0:
        if menor > L[i]:
            menor = L[i]
            posi = i
print(f'''Menor valor: {menor}
Posicao: {posi}''')

File: Java/Lista 4/lowestNumberPosition.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int x = sc.nextInt();
        int lista[] = new int[x];
        int p = 0;
        int me = 1000000;
        for (int n = 0; n < lista.length; n++){
            lista[n] = sc.nextInt();
            if (lista[n] < me){
                me = lista[n];
                p = n;
            }
        }
        System.out.println("Menor valor: " + me);
        System.out.println("Posicao: " + p);
    }
}